pkg/pillar/cmd/volumemgr/updatestatus.go

```diff
@@ -460,6 +464,13 @@ func doUpdateVol(ctx *volumemgrContext, status *types.VolumeStatus) (bool, bool)
 
 	// Anything to do?
 	if status.State == types.CREATED_VOLUME {
+		if ctx.hvTypeKube {
+			ctStatus := ctx.LookupContentTreeStatus(status.ContentID.String())
+			if ctStatus.IsContainer() {
+				log.Functionf("doUpdateVol: reference name %v, return changed", status.ReferenceName)
+				return true, false
```

Comment:
Why are you returning changed=true here? This function didn't modify the VolumeStatus hence no need to do that.

Reply:
Yeah removed that.